application/common/core.cpp

```diff
@@ -295,13 +337,40 @@ main(int argc, char *argv[])
 
     /* Main render loop */
     gLoomExecutionDone = 0;
+
+    /* Game Controller stuff */
+    SDL_GameControllerEventState(SDL_ENABLE);
+
+    //Adding non-existent controller mappings
+    int mappingsAdded = SDL_GameControllerAddMapping("4d6963726f736f66742050432d6a6f79,X360 Controller,platform:Windows,a:b10,b:b11,x:b12,y:b13,back:b5,start:b4,guide:b14,leftshoulder:b8,rightshoulder:b9,leftstick:b6,rightstick:b7,leftx:a0,lefty:a1,rightx:a2,righty:a3,lefttrigger:a4,righttrigger:a5,dpup:b0,dpleft:b2,dpdown:b1,dpright:b3,");
+    //int mappingsAdded = SDL_GameControllerAddMappingsFromFile("gamecontrollerdb.txt");
```

Comment:
Should we have a way to pass this from script so that e.g. end users can add more mappings?